src/amperity/gocd/agent/aurora/agent.clj

```diff
@@ -27,7 +31,9 @@
 (defn migrate-settings
   "Migrate an existing map of profile settings to the latest representation."
   [settings]
-  {:agent_tag (:agent_tag settings)
+  {:id (:id settings)
```

Comment:
It looks like this is both new and a required attribute; is it okay for it to be `nil` here for existing settings?

Reply:
This attribute is managed by/already required by GoCD so it shouldn't be `nil`.